frontend/main.js

```diff
@@ -103,19 +106,45 @@ async function startApp () {
     // Since a runtime error just occured, we likely want to persist app logs to local storage now.
     sbp('appLogs/save')
   }
+
+  await sbp('gi.db/settings/load', 'CHELONIA_STATE').then(async (cheloniaState) => {
+    // TODO: PLACEHOLDER TO SIMULATE CHELONIA IN A SW
+    if (!cheloniaState) return
+    const identityContractID = await sbp('gi.db/settings/load', SETTING_CURRENT_USER)
+    if (!identityContractID) return
+    Object.assign(sbp('chelonia/rootState'), cheloniaState)
+  })
+
+  const saveChelonia = () => sbp('okTurtles.eventQueue/queueEvent', 'CHELONIA_STATE', () => {
+    return sbp('gi.db/settings/save', 'CHELONIA_STATE', sbp('chelonia/rootState'))
+  })
+  const saveCheloniaDebounced = debounce(saveChelonia, 200)
```

Comment:
There is already a `debouncedSave` in `state.js`... how does this relate to it, and perhaps maybe this code should be moved to `state.js`?

Reply:
This is for Chelonia state specifically. With this PR, the state is broken down into Vuex state (that's `state.js`) and Chelonia state (that will later be in the SW).

This affects logging in, for example. When logging in, cheloniaState is saved in the SW and then Vuex state is broadcasted to all tabs.